lms/djangoapps/verify_student/services.py

```diff
@@ -199,7 +188,7 @@ def user_status(cls, user):
             return user_status
 
         user_status['should_display'] = attempt.should_display_status_to_user()
-        if attempt.created_at < earliest_allowed_verification_date():
+        if attempt.expiration_datetime < earliest_allowed_verification_date():
```

Comment:
I don't believe this is right. If you do this, for those verifications that are expired 1 day after the expiration date won't show as expired.
Please add a test case to catch this mistake.

Reply:
Basically, if you have an attempt created at 2019-11-08, and expires 2020-11-08, this condition won't hit. because the `earliest_allowed_verification_date()` will return 2019-11-09 for today. That is less than the `attempt.expiration_datetime`.

Reply:
I just pushed the changes for this. There was a flaw in the logic where `user_status` would never actually return 'expired', because it was only counting active verifications (so if the user only had expired verifications, it wouldn't return anything useful). Should be fixed now.